### DISCUSSION TITLE
Need free api

### DISCUSSION THREAD

**Discussion by sagark237** [community/community#139340] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

If some one can share an a free api for movie list banner and other details  which i can deploy on netlify

**Comment by Vineettiwari02** [comment, marked as answer]
Here you can get free API by https://www.themoviedb.org/ which you can deploy on Netlify I have also done this

**Comment by sagark237** [comment]
Thnaks @Vineettiwari02

**Comment by LiteBrite82** [comment]
Hi @sagark237 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
